src/lib/converter/ts-internal.ts

```diff
@@ -0,0 +1,130 @@
+import * as ts from "typescript";
+
+/**
+ * Expose the internal TypeScript APIs that are used by TypeDoc
+ */
```

Comment:
+1 to identifying the internal APIs so we can move off of them. Way to also link to the internal TS definitions.